routers/web/repo/setting.go

```diff
@@ -92,15 +93,16 @@ func Settings(ctx *context.Context) {
 		return
 	}
 	ctx.Data["PushMirrors"] = pushMirrors
+}
 
+// Settings show a repository's settings page
+func Settings(ctx *context.Context) {
 	ctx.HTML(http.StatusOK, tplSettingsOptions)
 }
 
 // SettingsPost response for changes of a repository
 func SettingsPost(ctx *context.Context) {
 	form := web.GetForm(ctx).(*forms.RepoSettingForm)
```

Comment:
Only two lines on `SettingsCtxData` will be reused. Is that necessary to extract `SettingsCtxData`?

Reply:
What do you mean? This isn't a refactor, other ctx data is now also being loaded on post request.

Reply:
There's actually a lot of things that should be being reused but aren't.